docs/tutorial/composition.md

```diff
@@ -0,0 +1,221 @@
+# Composition
+
+`concrete-python` supports circuit __composition__, which allows to use the output of a circuit execution, directly as an input, without the need to decrypt it in between. We can execute the circuit as many time as we want, by forwarding outputs, without decrypting intermediate values. This feature enables a new range of applications among which, a support for control flow in pure (cleartext) python.
```

Comment:
`concrete-python supports circuit __composition__, which allows to use the output of a circuit execution, directly as an input, without the need to decrypt it in between.` -> `concrete-python supports circuit __composition__, which allows the output of a circuit execution to be used directly as an input without decryption.`

`We can execute the circuit as many time as we want, by forwarding outputs, without decrypting intermediate values.` -> `We can execute the circuit as many times as we want by forwarding outputs without decrypting intermediate values.`

` This feature enables a new range of applications among which, a support for control flow in pure (cleartext) python.` -> ` This feature enables a new range of applications, including support for control flow in pure (cleartext) python.`

Reply:
Seems better this way yes.